### PR TITLE
Gentoo pip --user add

### PR DESCRIPTION
**Pull request by oscarandrea**
update of the previous request (https://github.com/JBakamovic/yavide/pull/59)

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -43,6 +43,7 @@ guess_system_package_manager(){
         SYSTEM_PACKAGE_TYPE="ebuild"
         SYSTEM_PACKAGE_MANAGER_INSTALL="emerge"
         SYSTEM_PACKAGE_MANAGER_UPDATE="emerge --sync"
+        PIP_INSTALL_CMD="--user"
```
Comment (JBakamovic) [inline install.sh]: This will work only for `emerge`-based distros. `PIP_INSTALL_CMD` must be defined for other cases as well.

File: install.sh [hunk 2]
```diff
@@ -164,7 +165,7 @@ stty $stty_orig     # restore terminal setting.
 #####################################################################################################
 echo "$passwd" | sudo -S $SYSTEM_PACKAGE_MANAGER_UPDATE
 echo "$passwd" | sudo -S $SYSTEM_PACKAGE_MANAGER_INSTALL $SYSTEM_PACKAGE_SET
-echo "$passwd" | sudo -S pip install $PIP_PACKAGE_SET
+echo "$passwd" | sudo -S pip install $PIP_INSTALL_CMD $PIP_PACKAGE_SET
```
Comment (JBakamovic) [inline install.sh]: This doesn't make sense. `pip install` should be part of `PIP_INSTALL_CMD`. In Gentoo case `PIP_INSTALL_CMD` shall be overridden and set to `pip install --user` .

**Comment by oscarandrea** [comment]
Ah is true, can I update the pull or should I make a new one?

**Comment by JBakamovic** [comment]
Just update this one.

**Comment by oscarandrea** [comment]
I updated, I hope it goes well.

File: install.sh [hunk 3]
```diff
@@ -18,32 +18,37 @@ guess_system_package_manager(){
         SYSTEM_PACKAGE_TYPE="rpm"
         SYSTEM_PACKAGE_MANAGER_INSTALL="dnf -y install"
         SYSTEM_PACKAGE_MANAGER_UPDATE="dnf --refresh check-update"
+        PIP_INSTALL_CMD="pip install "
```
Comment (JBakamovic) [inline install.sh]: Move this variable definition into the `Variables` section so we don't have to redefine it each time we add support for a new package manager.

File: install.sh [hunk 4]
```diff
@@ -18,32 +18,37 @@ guess_system_package_manager(){
...
     elif [ "`which apt-get`" != "" ]; then
         SYSTEM_PACKAGE_MANAGER="apt-get"
         SYSTEM_PACKAGE_TYPE="deb"
         SYSTEM_PACKAGE_MANAGER_INSTALL="apt-get -y install"
         SYSTEM_PACKAGE_MANAGER_UPDATE="apt-get update"
+        PIP_INSTALL_CMD="pip install "
```
Comment (JBakamovic) [inline install.sh]: Remove.

File: install.sh (hunk 3 above)
Comment (JBakamovic) [inline install.sh]: Remove extra space at the end.

File: install.sh [hunk 5]
```diff
@@ -18,32 +18,37 @@ guess_system_package_manager(){
...
         SYSTEM_PACKAGE_MANAGER_INSTALL="pacman --noconfirm -S"
         SYSTEM_PACKAGE_MANAGER_UPDATE="pacman -Syu"
+        PIP_INSTALL_CMD="pip install "
     elif [ "`which emerge`" != "" ]; then
         SYSTEM_PACKAGE_MANAGER="emerge"
         SYSTEM_PACKAGE_TYPE="ebuild"
         SYSTEM_PACKAGE_MANAGER_INSTALL="emerge"
         SYSTEM_PACKAGE_MANAGER_UPDATE="emerge --sync"
-        PIP_INSTALL_CMD="--user"
+        PIP_INSTALL_CMD="pip --user install "
```
Comment (JBakamovic) [inline install.sh]: Remove extra space at the end.

File: install.sh [hunk 6]
```diff
@@ -165,7 +170,7 @@ stty $stty_orig     # restore terminal setting.
 #####################################################################################################
 echo "$passwd" | sudo -S $SYSTEM_PACKAGE_MANAGER_UPDATE
 echo "$passwd" | sudo -S $SYSTEM_PACKAGE_MANAGER_INSTALL $SYSTEM_PACKAGE_SET
-echo "$passwd" | sudo -S pip install $PIP_INSTALL_CMD $PIP_PACKAGE_SET
+echo "$passwd" | sudo -S  $PIP_INSTALL_CMD $PIP_PACKAGE_SET
```
Comment (JBakamovic) [inline install.sh]: Remove extra space before $PIP_INSTALL_CMD

**Comment by oscarandrea** [comment]
I updated, I hope it goes well.

**Comment by JBakamovic** [comment]
Thanks for the change. It's merged into the master now.

**Pull request merged**